metaverse_modules/nameplate/index.js

```diff
@@ -0,0 +1,125 @@
+import * as THREE from 'three';
+import metaversefile from 'metaversefile';
+import {GLTFLoader} from 'three/examples/jsm/loaders/GLTFLoader.js';
+import {getModelGeoMat} from './model';
+import {Matrix4} from 'three';
+
+const {useApp, useCamera, useLocalPlayer, useMaterials, useFrame, useText} =
+  metaversefile;
+
+const Text = useText();
+const gltfLoader = new GLTFLoader();
+let nameplateMesh = null;
+
+async function createNameplateMesh() {
+  if (nameplateMesh) return;
+  const nameplateModelUrl = '/assets/nameplate.glb';
+  const nameplateModel = await new Promise((resolve, reject) => {
+    gltfLoader.load(nameplateModelUrl, resolve, () => {}, reject);
+  });
+  const {geometry, material} = getModelGeoMat(nameplateModel);
+  nameplateMesh = new THREE.InstancedMesh(geometry, material, 1000);
+}
+
+const createNameplateInstance = () => {
+  if (!nameplateMesh) return 0;
+  if (!nameplateMesh.instanceIndex) nameplateMesh.instanceIndex = 0;
```

Comment:
I think this variable is called `.count`?

https://threejs.org/docs/#api/en/objects/InstancedMesh.count

We do we need to declare it again?